pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md

```diff
@@ -0,0 +1,66 @@
+---
+$title: Publishing AMP Documents Checklist 
+$order: 0
+description: Responsive web design is about building fluid web pages that respond to your user's needs—pages that fit their device's screen size and orientation. You can achieve ...
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Follow this checklist to ensure your pages are getting all the benefits AMP has to offer by being healthy and accessible on the web! 
+
+
+# Ensure AMP Specification Validation 
+
+AMP comes with a ton of built in benefits, such as decreasing content load times by preloading content from AMP Caches. However, only documents that can be verified as valid AMP get these benefits. If you publish documents that have critical AMP errors your pages will not be indexed by an AMP cache, or possible served as error pages. 
+
+Learn more about the tools you can use to validate your AMP pages:
+
+
+
+*   [Validate AMP pages]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/validation-workflow/validate-amp.html', locale=doc.locale).url.path}}?format=websites)
+*   [The AMP Validator ](https://validator.ampproject.org/)
+*   [AMP Tools]({{g.doc('/content/amp-dev/documentation/documentation/tools.html', locale=doc.locale).url.path}}?format=websites) 
+
+
+# Grant cached AMP pages server access
+
+Valid AMP pages are automatically opted into all existing AMP caches. This is great news! Your pages are automatically preloaded efficiently and safely, enabling user-beneficial performance optimizations to content. However, documents served from AMP caches are served from domains that do not match your own. This can cause  Cross-Origin Resource Sharing (CORS) issues when using dynamic AMP components, such as [`<amp-form>`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-form.html', locale=doc.locale).url.path}}?format=websites) or [`<amp-list>`]({{g.doc('/content/amp-dev/components/reference/amp-list.html', locale=doc.locale).url.path}}?format=websites), causing your site to lose access to your data. This means certain content may not load correctly or at all.  
```

Comment:
* Mention that it's important to enable CORS Requests from all available AMP Caches: https://cdn.ampproject.org/caches.json
* Maybe link to the [AMP CORS lib](https://www.npmjs.com/package/amp-toolbox-cors) which implements all the best practices

Reply:
> Mention that it's important to enable CORS Requests from all available AMP Caches: cdn.ampproject.org/caches.json

Yea this is good to mention though I am not sure about the snippet about enabling it for all. Though yes that would be ideal and most correct. I would just be careful about the wording as enabling it for all is not a prerequisite for getting it to work. Maybe note that they should be mindful of the all the available AMP caches and to configure their servers to allow for the ones for their particular needs.